src/Elasticsearch/Client.php

```diff
@@ -1311,6 +1312,10 @@ public function extractArgument(&$params, $arg)
         }
     }
 
+    /**
+     * @param $name
+     * @param $var
+     */
     private function verifyNotNullOrEmpty($name, $var)
     {
         if ($var === null) {
```

Comment:
I don't understand what's the problem here.

Besides: 
> If you rely on strcmp for safe string comparisons, both parameters must be strings, the result is otherwise extremely unpredictable.
For instance you may get an unexpected 0, or return values of NULL, -2, 2, 3 and -3.

(from the top comment on http://php.net/manual/fr/function.strcmp.php)